packages/core/src/index.ts

```diff
@@ -146,6 +138,7 @@ export class Microfleet extends EventEmitter {
   public readonly [constants.CONNECTORS_PROPERTY]: StartStopTree
   public readonly [constants.DESTRUCTORS_PROPERTY]: StartStopTree
   public readonly [constants.HEALTH_CHECKS_PROPERTY]: PluginHealthCheck[]
+  private connectorToPlugin: Map<any, string>
```

Comment:
`connectorsToPlugins` might be more expressive, and maybe it's type could be better defined as `Map<PluginConnector, string>